app/views/register/suitability/BeforeYouContinueTaxableAgentView.scala.html

```diff
@@ -48,11 +50,11 @@
         <p>@messages("suitability.beforeYouContinue.taxable.agent.p4")</p>
         <p>@messages("suitability.beforeYouContinue.taxable.agent.p5")</p>
 
-        <ul>
+        <ul class="govuk-list">
```

Comment:
Please add  ` govuk-list--bullet govuk-list--spaced` to this  <ul class="govuk-list"> to address accessibility issues Andy has brought up around lists of links - and do it in the other BeforeYourContinue views where the links are

Reply:
fixed now